src/poliastro/twobody/orbit.py

```diff
@@ -1024,6 +1024,27 @@ def propagate(self, value, method=mean_motion, rtol=1e-10, **kwargs):
                 new_epoch,
             )
 
+    @u.quantity_input(value=u.rad)
+    def time_to_anomaly(self, value):
+        """ Returns time required to be in a specific true anomaly.
+
+        Parameters
+        ----------
+        value : ~astropy.units.Quantity
+
+        Returns
+        -------
+        tof: ~astropy.units.Quantity
+            Time of flight required.
+        """
+
+        # Compute time of flight for correct epoch
+        M = nu_to_M(self.nu, self.ecc)
+        new_M = nu_to_M(value, self.ecc)
+        tof = Angle(new_M - M).wrap_at(360 * u.deg) / self.n
```

Comment:
Can we reuse this logic in `propagate_to_anomaly`? We could do

```
time_of_flight = self.time_to_anomaly(value)
```

and drop lines 1062 to 1071.

Reply:
I simplified the `propagate_to_anomaly` by making use of this new method and also using `self.property` instead of calling the `rv2coe` function, which lead to dropping those lines :+1: